### DISCUSSION TITLE
Problem in the useInfiniteQuery

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#130428] [Programming Help]
### Select Topic Area

Question

### Body

Hey, can someone help with the following code shown. After deploying the website, it is showing certain errors. I tried resolving the issue but didn't helped much. I am attaching the GitHub Link to the code as well.

<img width="1456" alt="Screenshot 2024-06-29 at 15 55 13" src="https://github.com/community/community/assets/91900978/845c77c9-c6d1-4f4f-a798-264cbf7c5abe">

<img width="1792" alt="Screenshot 2024-06-29 at 15 56 07" src="https://github.com/community/community/assets/91900978/8bf0c875-46e9-480a-8027-f3223a6b374f">

<img width="1792" alt="Screenshot 2024-06-29 at 15 56 58" src="https://github.com/community/community/assets/91900978/2524a08b-b55f-4199-9cd7-b2526d43c5d7">

GitHub Repo Link: https://github.com/iLakshya/Snapgram

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @iLakshya ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
